DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/_SectionProgress.cshtml

```diff
@@ -0,0 +1,59 @@
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateProgress
+@model SectionProgressViewModel
+
+@{
+  var positiveAttempts = Model.Attempts.HasValue && Model.Attempts > 0;
+}
+
+<div>
+  <div class="basic-summary-list nhsuk-u-margin-bottom-3">
+    <div class="nhsuk-grid-column-full nhsuk-body-l nhsuk-u-font-weight-bold word-break">
+      @Model.SectionName
+    </div>

```

Comment:
Looking at the screen shots, the section names seem slightly more indented than the details underneath them, would be nice to have them all inline.